src/app/components/TutorialDialog/container/index.tsx

```diff
@@ -0,0 +1,79 @@
+// TUTORIALDIALOG CONTAINER
+// Logic for if dialog should render
+
+import React, { useState, useEffect, useCallback } from 'react';
+import { useIsConnected } from 'app/hooks/useAccount';
+import { Sovryn } from 'utils/sovryn';
+import { TutorialDialogComponent } from '../component';
+import { currentChainId } from 'utils/classifiers';
+
+export function TutorialDialog() {
+  //Check if previously connected, currently connected to RSK, currently wallet is connected, closed before
+  const [show, setShow] = useState<boolean>(false);
+
+  const onNetwork =
+    window.ethereum && parseInt(window.ethereum.chainId) === currentChainId;
+
+  const checks = {
+    // previousUser:
+    //   window.localStorage.getItem('connectedToRskBefore') === 'true',
+    connected: useIsConnected(),
+    closedBefore: window.sessionStorage.getItem('closedRskTutorial') === 'true',
```

Comment:
Instead of `window.sessionStorage` use `react-localstorage` package (should be already installed to our project).

Safari browser (at least older versions) doesn't have window.sessionStorage, window.localStorage or window.cookie objects when browsing in incognito mode so dapp will crash - this requires special handling and package will solve it.